Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/SyncPerformanceIntTests.swift

```diff
@@ -0,0 +1,95 @@
+import XCTest
+@testable import MongoSwift
+import StitchCore
+import StitchCoreSDK
+import StitchCoreAdminClient
+import StitchDarwinCoreTestUtils
+@testable import StitchCoreRemoteMongoDBService
+import StitchCoreLocalMongoDBService
+@testable import StitchRemoteMongoDBService
+
+import Foundation
+
+class SyncPerformanceIntTests: BaseStitchIntTestCocoaTouch {
```

Comment:
[q] I thought we decided we didn't want to inherit from BaseStitchIntTestCocoaTouch. What changed? This is part of why you're getting errors when the local server is not running for prod tests.